build-debezium-multiplatform.sh

```diff
@@ -6,6 +6,30 @@ if [ -z "${DEBEZIUM_DOCKER_REGISTRY_PRIMARY_NAME}" ]; then
   DEBEZIUM_DOCKER_REGISTRY_PRIMARY_NAME=quay.io/debezium
 fi;
 
+# Function to convert release tag to semver
```

Comment:
I'd extract this function into separate file and include it here and in `build-debezium.sh` to avoid repeating the code/